### ISSUE TITLE
`chalice package` does not create pure lambda resources in sam.json

### ISSUE THREAD

**Issue by lawrlee** [aws/chalice#705]
I'm trying to move my chalice application to a CI/CD environment where `chalice deploy` isn't optimal. However, my application has a number of pure lambda functions that are integrated with s3 triggers. When I call `chalice package`, the resulting `sam.json` only contains resources for my `app.route()` functions. Is this a bug?

**Comment by kyleknap** [comment]
Yeah it is a bug. The `chalice package` command is missing the cloudwatch events (e.g. `app.cron()` and `app.rate()`) specific lambda functions as well. So we should add that as well.

**Comment by djn658** [comment]
Is this gonna be fixed anytime soon?

**Comment by stealthycoin** [comment]
@djn658 It resolved with the new deployer that just got merged into master. You can use the master branch to test it out if you like, and this issue will be closed when the new release is cut for version 1.2.0 which should be pretty soon.

**Comment by djn658** [comment]
@stealthycoin Thank you very much for your reply! This is going to be very helpful.

**Comment by wutachih** [comment]
Can I include native lambda handler in app.py? Or I need to use domovoi.

**Comment by BigChief45** [comment]
@stealthycoin I haven't tried this yet but if it's resolved in 1.2.0 I think this issue can be closed now.

**Comment by jamesls** [comment]
Yep, this should be fixed now.  @wutachih you can use a pure lambda function in your app.py now and chalice will properly generate a SAM template as of 1.2.0:

```python

app = Chalice(app_name='foo')


@app.lambda_function()
def foo(event, context): pass
```
